src/mixins/object_interactivity.mixin.js

```diff
@@ -142,8 +142,30 @@
       if (!dimensions) {
         dimensions = this._getNonTransformedDimensions();
       }
-      var transformMatrix = this._calcDimensionsTransformMatrix();
-      return fabric.util.transformPoint(dimensions, transformMatrix, true);
+      var points = [
+          {
+            x: -dimensions.x/2,
```

Comment:
maybe is better to cache the dimensions and matrix with the combinatio of skew and scale.
something like we do with font width caching and beizer to arc caching.

but as of now is an early optimization. help me to understand why it fails minifing